src/flatten_dict/splitter.py

```diff
@@ -10,6 +10,11 @@ def path_splitter(flat_key):
     return keys
 
 
+def dot_splitter(flat_key):
+    keys = flat_key.split('.')
```

Comment:
Please refer to `underscore_splitter` below. `tuple` generally will have better performance in the later steps.